.github/workflows/ci.yml

```diff
@@ -182,7 +182,7 @@ jobs:
       - name: Run ${{ matrix.test }}
         run: |
           # shellcheck disable=SC2046
-          go test -v -race -tags integration -run ${{ matrix.test }} $(git grep -l '^//go:build integration' | xargs -I {} dirname './{}' | sort | uniq)
+          go test -v -race -tags integration -run ${{ matrix.test }} ./...
```

Comment:
Yep, as mentioned in the other comment this one should stay.

Reply:
Line 159 is the one I was thinking of, but it actually looks like that's already doing the correct thing, so disregard!